pkg/daemon/ceph/client/filesystem.go

```diff
@@ -174,6 +175,13 @@ func IsMultiFSEnabled() bool {
 	}
 	return false
 }
+func SetPools(context *clusterd.Context, cephVersion cephver.CephVersion, clusterName, fsName string, dataPool []*model.Pool, metdataPool *model.Pool) error {
+	args_metadata := []string{"osd", "pool", "set", fsName + "-metadata", "size", strconv.Itoa(int(metdataPool.ReplicatedConfig.Size))}
+	NewCephCommand(context, clusterName, args_metadata).Run()
+	args_dataPool := []string{"osd", "pool", "set", fsName + "-data0", "size", strconv.Itoa(int(dataPool[0].ReplicatedConfig.Size))}
+	NewCephCommand(context, clusterName, args_dataPool).Run()
+	return nil
```

Comment:
`SetPoolProperty()` will return an error that you should return from this method